ikabot/function/buyResources.py

```diff
@@ -91,6 +91,8 @@ def getOffers(session, city):
         #Parse CDN Images to material type
         if offer["tipo"] == '//gf2.geo.gfsrv.net/cdn19/c3527b2f694fb882563c04df6d8972':
              offer["tipo"] = 'wood'
+        elif offer["tipo"] == '//gf1.geo.gfsrv.net/cdnc6/94ddfda045a8f5ced3397d791fd064':
+            offer["tipo"] == 'wine'     
```

Comment:
I think there should be a single equal here, not a double. `offer["tipo"] = 'wine' `